### ISSUE TITLE
[Question] Difference with 'Redirector' by Einar Egilsson

### ISSUE THREAD

**Issue by ghost** [lifo9/Redirector#1]
Hi,

I discover your 'Redirector' on AMO at https://addons.mozilla.org/en-US/firefox/addon/my-redirector/.
I've been using 'Redirector' by Einar Egilsson for years :
AMO : https://addons.mozilla.org/en-US/firefox/addon/redirector/
GitHub : https://github.com/einaregilsson/Redirector

I see on the video available here that both 'Redirectors' are very similar, graphically speaking.

I just wish to know the differences between the two.

Thanks!

**Comment by lifo9** [comment]
Hi Cade66!

The addon you mentioned needs a whole bunch of premissions:
![image](https://user-images.githubusercontent.com/40486100/192320295-5ad76fbd-9396-40d7-839b-7bab2e2a64d8.png)

My addon doesn't need `Access your data for all websites`, but requests just required permissions based on rules, instead. I'm not saying that the addon you mentioned does someting malicious, but I like more granular aproach to permissions better.
Other than that, I can't see much of a difference, but I didn't try it.
Honestly, I just wanted something that would suite my needs and didn't want to use some 3rd party addon, which source code I didn't review.

Feel free to try it out, I'll be glad for any feedback :)

**Comment by Cade66** [comment]
I've installed it, unfortunately it doesn't import Einar Egilsson's 'Redirector''s exported rules. I have over 50 of them, some quite elaborated, so re-writing them all faces time and using both faces redundancy. Another time maybe, unless an update handles importing rules set for the first to your 'Redirector'.
Hence I've proceeded to no comparison both. I guess from what you wrote above that besides the required permissions they have similar features..
